did/stats.py

```diff
@@ -213,6 +213,12 @@ def configured_plugins(self, config):
                 else:
                     items_created = True
 
+            # public-inbox is more user friendly but it wouldn't be
+            # possible to import it, so let's just create an alias when
+            # looking up the plugin type.
+            if type_ == 'public-inbox':
+                type_ = 'public_inbox'
```

Comment:
What about just substituting `-` for `_` in general? There might be other multi-word plugins in the future.